### DISCUSSION TITLE
Welcome to to-do-app Discussions!

### DISCUSSION THREAD

**Discussion by Hassib1234** [CSproMates/to-do-app#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samimham2019** [comment]
thank stay tuned

**Comment by Abhay2342** [comment]
What is this app about?

**Reply by samimham2019** [reply]
Dear, it's just for practice sake in order for us to learn how to use github for project management among team the HTML and js are the apps main files represents the whole structure of to do app while the others are merged as practice.

**Reply by Abhay2342** [reply]
Oh, if there is something that I can help with, please let me know.
